apps/web/src/components/FormContent.tsx

```diff
@@ -0,0 +1,39 @@
+import { FormStepHeader }  from '@components';
+import {Profile}  from '../pages/forms/Profile';
+import {CareActivitiesBundle} from '../pages/forms/CareActivitiesBundle';
+import {Occupation} from '../pages/forms/Occupation';
+import {ActivitiesGap} from '../pages/forms/ActivitiesGap';
+import {Suggestions} from '../pages/forms/Suggestions';
+
+interface FormContentProps {
+  step: number;
+  formTitle: string;
+}
+
+export const FormContent: React.FC<FormContentProps> = ({step, formTitle}) => {
```

Comment:
Rename this to PlanningContent or PlanningStepWrapper

Reply:
ok done